### DISCUSSION TITLE
Where can I map my midi keyboard?

### DISCUSSION THREAD

**Discussion by heyudude** [GillesLACAUD/OPLA-Source-code#58] [General]
Hi,

Opla works fine. Thanks for the prompt shipment!

Where can I map my midi keyboard to control the settings?

**Comment by GillesLACAUD** [comment]
Here you can download the mapping for an Arturia Mini Lab MK2
https://github.com/GillesLACAUD/OPLA-OTHER-FILES

**Reply by heyudude** [reply]
Thanks
There are 8 of them?
Or just use the latest?

And how to use the mapping? Compile and upload?

**Reply by GillesLACAUD** [reply]
You have eight of them...yes
Just check you to use it with the Arturia control center software

**Comment by GillesLACAUD** [comment]
Or you can open this file for the complete MIDI mapping
Or you can open the documentation on the web site
[https://0966bfcf-320f-4523-82de-e42c2f275251.filesusr.com/ugd/ded0dd_67508e536d1e45589b7eaefd604f1c2a.pdf](url)
Note: Each time you use the virtual slider you have the midi CC number associate with the parameter on the display

Musically
Gilles/Hansy Synth
